### PR TITLE
[full-ci] fix: render loading state before loading starts

### PR DESCRIPTION
**Pull request by kulmann**
## Description
Noticed that the `Personal` view showed the `Resource not found` component for a brief moment before the resource loader task was started. This is due to the task API not announcing itself as `isRunning=true` when there is no task instance, yet. Fixed in all views by adding a check for the absence of a loader task instance (in addition to the `isRunning` check).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortraitNotifications</strong> https://drone.owncloud.com/owncloud/web/27007/44/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by dschmidt — APPROVED** [review]

File: packages/web-runtime/tests/unit/components/SidebarNav/SidebarNavItem.spec.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ const propsData = {
   active: false,
   target: exampleNavItem.route.path,
   icon: exampleNavItem.icon,
-  index: 5,
+  index: '5',
```
Comment (lookacat) [inline packages/web-runtime/tests/unit/components/SidebarNav/SidebarNavItem.spec.js:24]: how is this related? 🤔

Comment (kulmann) [reply packages/web-runtime/tests/unit/components/SidebarNav/SidebarNavItem.spec.js:24]: It's not. It just printed a warning to the console when running unit tests, so I fixed it. ;-)
